### DISCUSSION TITLE
How to add Pinterest to social-share.html

### DISCUSSION THREAD

**Discussion by lsolesen** [mmistakes/minimal-mistakes#2562] [Q&A] [answered]
Has any of you added Pinterest to the social-share.html so it looks like the other buttons?

**Comment by mmistakes** [comment, marked as answer]
See the docs. I cover how to add other social networks.

https://mmistakes.github.io/minimal-mistakes/docs/layouts/#social-sharing-links

**Comment by lsolesen** [comment]
@mmistakes  When adding it - it ends up looking like this.

![Screenshot (16)](https://user-images.githubusercontent.com/148026/84594496-ea51c480-ae52-11ea-9e8a-f91e82715aea.png)

See  -- https://github.com/motionsplan/motionsplan-dk-jekyll/blob/pinterest/_includes/social-share.html

**Comment by mmistakes** [comment]
Did you add Pinterest button styles? I mention how to do that in the link above using a Reddit button as an example.

**Comment by lsolesen** [comment]
Figured it out. You need to add `data-pin-custom="true"` to the link.
